src/textual/cache.py

```diff
@@ -34,6 +29,12 @@ class LRUCache(Generic[CacheKey, CacheValue]):
 
     Each entry is stored as [PREV, NEXT, KEY, VALUE] where PREV is a reference
     to the previous entry, and NEXT is a reference to the next value.
+
+    Note that stdlib's @lru_cache is implemented in C and faster! It's best to use
+    @lru_cache where you are caching things that are fairly quick and called many times.
+    Use LRUCache where you want increased flexibility and you are caching slow operations
+    where the overhead of the cache is a small fraction of the total processing time.
+
```

Comment:
```suggestion
```

Reply:
What have you got against blank lines??

Reply:
Trailing empty blank lines immediately after closing a docstring look _so_ ugly.

If you ask, I'll stop deleting them.
But they look so ugly.